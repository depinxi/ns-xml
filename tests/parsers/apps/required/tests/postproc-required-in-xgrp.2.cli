-A "Same as postproc-required-in-xgrp.1.cli" -C "but the --multi option" --multi will activate -b 'the xgroup'