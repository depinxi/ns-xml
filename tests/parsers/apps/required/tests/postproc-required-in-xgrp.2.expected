CLI: "-A", "Same as postproc-required-in-xgrp.1.cli", "-C", "but the --multi option", "--multi", "will", "activate", "-b", "the xgroup"
Value count: 0
Values: 
Error count: 0
Subcommand: 
rargumentA=Same as postproc-required-in-xgrp.1.cli
groupS=
argumentC=but the --multi option
groupT=multiargC
switchB=False
argumentE=
switchI=False
rargumentF=
multiargC=will activate
groupU=
groupV=
argumentF=
multiargG=
switchH=False
argumentNumber=
argumentNumberPositive=
groupQ=
switchA=False
argumentB=the xgroup
